contracts/PodCastNFT.sol

```diff
@@ -51,15 +66,44 @@ contract PodCastNFT is ERC721, ConsensusAdminable {
             super.supportsInterface(interfaceId);
     }
 
+    function getSVG() public pure returns (string memory) {
+        return Card.svg();
+    }
+
     function mintAndTransfer(address _to) public returns (uint256) {
         require(
             hasRole(ADMIN_ROLE, msg.sender),
             "You are not authorized to mint"
         );
-        _tokenIds.increment();
 
+        _tokenIds.increment();
         uint256 newItemId = _tokenIds.current();
+
+        string memory svg = getSVG();
+        string memory json = Base64.encode(
+            bytes(
+                string(
+                    abi.encodePacked(
+                        '{"name": ',
+                        '"Henkaku PodCast NFT",',
+                        '"description": ',
+                        '"Henkaku PodCast NFT prototype",',
+                        '"image": "',
+                        svg,
+                        '","image_data": "',
```

Comment:
you don't need this. only supported in Opensea and if you have image then you don't need the one.
if you omit this i think tx limited size will be decrease and should function

Reply:
@geeknees 
if you encounter errors, can you paste it here ?

Reply:
error is here. (I removed image_data.)

```sh
❯ npx hardhat run scripts/mint.js --network rinkeby
ProviderError: max code size exceeded
    at HttpProvider.request (/Users/masumi/ghq/github.com/henkaku-center/podcast-nft/node_modules/hardhat/src/internal/core/providers/http.ts:49:19)
    at LocalAccountsProvider.request (/Users/masumi/ghq/github.com/henkaku-center/podcast-nft/node_modules/hardhat/src/internal/core/providers/accounts.ts:187:34)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at EthersProviderWrapper.send (/Users/masumi/ghq/github.com/henkaku-center/podcast-nft/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
```

Reply:
Compilation succeeds, and fails to deploy the contract. As an addition, you need to change the following to your real (dev) address.

https://github.com/henkaku-center/podcast-nft/blob/61608368e828791c397ec39491bdbd4892ee398f/scripts/mint.js#L11

Reply:
I'll give it a try. But even if I solve this problem, I think the following problems will remain.

> 外部ファイル読み込みのSVGはOpenSeaは対応していない

Reply:
It was not enough to reduce imports, so I had to remove other logic to successfully deploy it. You can see the result of Deploy -> Mint in the following PR.

https://github.com/henkaku-center/podcast-nft/pull/6

The images still don't seem to render properly.

Reply:
> The images still don't seem to render properly.

you also need to encode svg by base64 
https://github.com/SamuraiT/dynamic-nft-sample/blob/main/contracts/DynamicNFT.sol#L51

Reply:
That's definitely the wrong place...my apologies. I fixed it, but the image doesn't show up.

Reply:
uh with encoded svg, you need to add `data:image/svg+xml;base64,` to indicate it's svg
https://github.com/SamuraiT/dynamic-nft-sample/blob/main/contracts/DynamicNFT.sol#L50

Reply:
Oh, I didn't complete my comment.

I think it has been done. If you paste the above code into your browser, it will be attached.

```
data:application/json;base64,eyJuYW1lIjogIkhlbmtha3UgUG9kQ2FzdCBORlQiLCJkZXNjcmlwdGlvbiI6ICJIZW5rYWt1IFBvZENhc3QgTkZUIHByb3RvdHlwZSIsImltYWdlIjogImRhdGE6aW1hZ2Uvc3ZnK3htbDtiYXNlNjQsUEhOMlp5QmxibUZpYkdVdFltRmphMmR5YjNWdVpEMG5ibVYzSURBZ01DQXlNREF3SURJd01EQW5JSFpwWlhkQ2IzZzlKekFnTUNBeU1EQXdJREl3TURBbklIaHRiRzV6UFNkb2RIUndPaTh2ZDNkM0xuY3pMbTl5Wnk4eU1EQXdMM04yWnljK1BHTnNhWEJRWVhSb0lHbGtQU2RoSno0OFkybHlZMnhsSUdONFBTYzJPVEl1TkNjZ1kzazlKekV4TnprdU55Y2djajBuTVRJeExqUW5MejQ4TDJOc2FYQlFZWFJvUGp4emQybDBZMmcrUEdadmNtVnBaMjVQWW1wbFkzUXZQanhuUGp4cGJXRm5aU0JvWldsbmFIUTlKekl3TURBbklIZHBaSFJvUFNjeU1EQXdKeUJvY21WbVBTZG9kSFJ3Y3pvdkwyZGhkR1YzWVhrdWNHbHVZWFJoTG1Oc2IzVmtMMmx3Wm5NdlVXMVZSVlJ6VlhveFkxSTJjR2s1VEZCNVJVWnVSM0pqZDFGWmIwZDBkbHB0UkdSNldqSkhaRXcwYTJaM2RpY3ZQanhuSUdacGJHdzlKeU5tWm1ZblBqeHdZWFJvSUdROUoyMDBOakl1TkNBeE1EWTNMamtnTXk0MkxUSXVNeTB1TnkweExqRWdOQzB5TGpZdU55QXhMakVnTXk0NUxUSXVOUzB1TnkweExqRWdOQzB5TGpZdU55QXhMakVnTXk0MUxUSXVNeUF4TGpjZ01pNDJMVE11TlNBeUxqTWdNUzQ0SURJdU9DMDBMakVnTWk0M0xqVXVOeUEyTGpRdE5DNHlJREV1T0NBeUxqaGpNUzR5SURFdU9DNDJJRFF1TWkweExqRWdOUzR6YkMwekxqSWdNaTR4TGpVdU55QTNMall0TkM0NUlERXVOeUF5TGpZdE55NDJJRFF1T1NBeExqY2dNaTQxTFRNdU55QXlMalF0TVM0M0xUSXVOUzAzTGpZZ05DNDVMVEV1TnkweUxqWWdOeTQyTFRRdU9TMHVOUzB1TnkwMkxqVWdOQzR5TFRNdU9TMDJJRFl1TlMwMExqSXRMalV0TGpjdE5DNHlJREl1TnkweExqZ3RNaTQ0TFRNdU5pQXlMak42YlRFeExqWWdOaTR5SURJdU5TMHhMall0TGpZdExqa3RNaTQxSURFdU5ucHRMVEV1T1MwM0xqa2dNeTQ1TFRJdU5TMHVOQzB1Tmkwekxqa2dNaTQxZW0wNElETXVPU0F5TGpRdE1TNDFMUzQyTFM0NUxUSXVOQ0F4TGpWNmJTMHlOQzR6TFRJekxqVXRNeTQwTGpKakxqTWdNUzQwSURBZ015NHhMVEVnTkM0NWJETXVOQ0F4TGpKakxqa3RNUzQ1SURFdU15MHpMamtnTVMwMkxqTjZiVEUxTGpJZ01TNDRZeTQwTFRFdU1pNDJMVEV1Tnk0MkxUSXVNMnd0TVM0ekxURXVPUzAzTGpRZ05DNDRMakl0TVM0eUlESXRNUzR6WXk0MUxTNDBMamd0TGprdU9TMHhMalZ6TFM0eExURXVNaTB1TkMweExqZHNMVEl1TlMwekxqa2dOUzR6TFRNdU5DMHhMamN0TWk0MkxUY3VPU0ExTGpFdExqa3RNUzQwTFRNdU1TQXlJQzQ1SURFdU5DMDNMallnTlNBeExqY2dNaTQySURVdU5pMHpMalpqTGpnZ01TNHhJREV1TWlBeUlERXVOQ0F5TGprdU1pQXhMakV1TVNBeUxqTXRMalFnTXk0MGJESXVPQzQwWXkwdU1pNDNMUzQxSURFdU5pMHhJREl1TlMwdU9DQXhMalF0TVM0M0lESXVOeTB5TGpVZ015NDFiREV1TnlBeUxqWmpMakl0TGpFZ01TNDBMVEV1TVNBeUxqSXRNaTR5TGpjdE1TQXhMak10TWlBeExqTXRNaTR4TGpndU1TQXhMall1TWlBeUxqVXVNUzB1TkM0MkxURXVNU0F4TGpVdE1pNHpJREl1TmkweExqSWdNUzB5TGpFZ01TNDNMVE11TWlBeUxqUnNNUzQ0SURJdU9HTXhMalV0TVNBekxqZ3RNaTQ1SURVdU1pMDBMak1nTVM0ekxURXVOQ0F5TGpRdE1pNDNJRE10TkNBekxqUXRMaklnTnkweExqa2dNVEF1TXkwekxqbHNMVEV1T1Mwell5MHlMamNnTVM0M0xUUXVOeUF5TGpZdE5pNHpJREl1T1M0eUxTNDFMalV0TVM0MElERXRNaTQzZW0wdE1USXVOeTAxTGprZ01TNDVMVEV1TWlBeExqVWdNaTR6Y3k0MUxqY3RMaklnTVM0eWJDMHhMak11T1NBeUxqUWdNaTR5TFRFdU9TNDNZekF0TVM0eUxTNDRMVE11TWkweUxqUXROaTR4ZW0wNExqTWdPQzR5WXkwdU15NDNMUzQySURFdU1pMHVPU0F4TGpRdExqa3VNeTB4TGprdU15MHpJQzR4YkRRdU5DMHlMamhqTUNBdU1pMHVNaTQzTFM0MUlERXVNM3B0TFM0MkxURXlMall0TVM0eklETXVNbU11TmlBd0lERXVOUzR5SURJdU9DNDNJREV1TVM0eklERXVPUzQzSURJdU5TQXhMakZzTVM0eUxUTXVOR010TGpZdExqTXRNUzQwTFM0MkxUSXVOUzB4TFRFdU15MHVOQzB5TGpFdExqWXRNaTQzTFM0MmVtMHROVFV1TWlBMk1TNHpZeTB5TGpRZ01TNDFMVE11T1NBekxqY3ROQzQySURZdU15MHVOeUF5TGpjdExqTWdOUzR4SURFdU1pQTNMalJzTnk0eUlERXhMaklnTkM0NExUTXVNUzB5TGpndE5DNHpJRGN1TmkwMUlESXVPQ0EwTGpNZ05DNDRMVE11TVMwM0xqSXRNVEV1TW1NdE1TNDBMVEl1TWkwekxqVXRNeTQyTFRZdU1pMDBMakV0TWk0M0xTNDBMVFV1TXk0eExUY3VOaUF4TGpaNmJURXdMalVnT0MwM0xqWWdOUzB4TGpVdE1pNDBZeTB1TmkweExTNDRMVEl0TGpVdE15NHljekV0TWk0eElESXVNUzB5TGpnZ01pNHlMUzQ1SURNdU5DMHVOeUF5TGpFdU9DQXlMamNnTVM0NGVtMHhPQzQwTFRJMkxqRXROaTQ1SURRdU5DMHVOeUF4TVMwMExqZ3ROeTQwTFRRdU9DQXpMakVnTVRJdU1pQXhPQzQ1SURRdU9DMHpMakV0TlM0ekxUZ3VNaUF4TVM0M0lEUXVNU0EyTGprdE5DNDFMVEUwTGpZdE5DNDFlbTB4TkM0NExUa3VOaUEzTGpJZ01URXVNbU11TmlBeElDNDRJRElnTGpVZ015NHljeTB4SURJdU1TMHlMakVnTWk0NExUSXVNaTQ1TFRNdU5DNDNMVEl1TVMwdU9DMHlMamN0TVM0NGJDMDNMakl0TVRFdU1pMDBMamdnTXk0eElEY3VNaUF4TVM0eVl6RXVOQ0F5TGpJZ015NDFJRE11TmlBMkxqSWdOQzR4Y3pVdU15MHVNU0EzTGpZdE1TNDJZekl1TkMweExqVWdNeTQ1TFRNdU55QTBMall0Tmk0ekxqY3RNaTQzTGpNdE5TNHhMVEV1TWkwM0xqUnNMVGN1TWkweE1TNHllbTB0TVRBdU1pQXhMalF0Tmk0M0xURXdMalFnTVRVdU5DQTBMamNnTXk0NExUSXVOQzB4TVM0MUxURTNMamd0TVRJekxqVXRNVEF1TkNBMk55NHlJREV3TXk0NExqa3RNVEl1TXlBMUxqUWdPQzR6SURRdU9DMHpMakV0TlM0ekxUZ3VNaUF4TVM0M0lEUXVNU0EyTGprdE5DNDFMVEUwTGpZdE5DNDFJREV1TmkweE15NDRMVFl1T1NBMExqUXRMamNnTVRFdE1pNHhMVE11TXlBeUxqRXRNamd1TnlBM0xqUWdNVEV1TlNBMExqY3RNeTR4TFRRdU5pMDNMaklnTnk0M0xUVWdOQzQySURjdU1pQTBMamN0TXk0eExURXhMall0TVRjdU9TQTBMall0TXlBeE1TNDJJREUzTGprZ01UTXVNUzA0TGpVdE1pNDVMVFF1TlMwNExqTWdOUzQwTFRFdU9DMHlMamNnT0M0ekxUVXVOQzB5TGprdE5DNDFMVGd1TXlBMUxqUXRNUzQ0TFRJdU55QTRMak10TlM0MExUSXVNeTB6TGpVZ05DNDFMVEl1T1NBeE1TNDFJREUzTGpoNmJUSXVNeTB4TXk0MExURTBMVFF1TXlBM0xqa3ROUzR4ZW0wdE5ERXVOU0F5TWk0ekxUY3VOeUExTFRRdE5pNHhkaTB1TW13M0xqWXROQzQ1ZWljdlBqeHdZWFJvSUdROUoyMHhNREU1TGpRZ05qRTVMamN0TlM0M0xUZ3VPQzA1TGpRZ05pNHhJRFV1TnlBNExqZ3ROQzQwSURJdU9DMHhNeTQyTFRJd0xqa2dOQzQwTFRJdU9DQTFMalVnT0M0MUlEa3VOQzAyTGpFdE5TNDFMVGd1TlNBMExqUXRNaTQ0SURFekxqWWdNakF1T1hvbkx6NDhjR0YwYUNCa1BTZHRNVEEwTVM0MUlEWXdOUzQwTFRFekxqa2dPUzB4TXk0MkxUSXdMamtnTVRNdU9TMDVJREl1TXlBekxqWXRPUzQxSURZdU1TQXpMak1nTlNBNExqa3ROUzQ0SURJdU1pQXpMalF0T0M0NUlEVXVPQ0F6TGpRZ05TNHpJRGt1TlMwMkxqRjZKeTgrUEhCaGRHZ2daRDBuYlRFd05EUXVPU0EyTURNdU1pMHhNeTQyTFRJd0xqa2dNeTQzTFRJdU5DQXhPQzQySURjdU15NHhMUzR4TFRndU9DMHhNeTQySURRdU1pMHlMamNnTVRNdU5pQXlNQzQ1TFRNdU5pQXlMak10TVRndU55MDNMalF0TGpFdU1TQTRMamtnTVRNdU4zb25MejQ4Y0dGMGFDQmtQU2R0TVRBMk5pNDBJRFU0T1M0ekxURXpMall0TWpBdU9TQTBMalF0TWk0NElEWXVNaUE1TGpVdU1TMHVNU0F4TGpjdE1UUXVOaUEwTGprdE15NHlMVEV1TnlBeE5DNHpJREUxTGpnZ05pNHpMVFV1TWlBekxqUXRNVEV1T1MwMExqY3RMalFnTXk0MklEUXVNU0EyTGpSNkp5OCtQSEJoZEdnZ1pEMG5iVEV3T1RVdU55QTFOak11TVMwM0xqUWdOQzQ0SURFdU55QTJMakV0TkM0MUlESXVPUzAyTGpNdE1qVXVOaUExTGpFdE15NHpJREl3TGpnZ01UWXVNaTAwTGpjZ00zcHRMVGd1TlM0NUlEVXVOUzB6TGpVdE9DNHlMVFl1T0MwdU1TNHhlaWN2UGp4d1lYUm9JR1E5SjIweE1UQTNMalFnTlRZeUxqZ3RNVE11TmkweU1DNDVJRFF1TkMweUxqZ2dOaTR5SURrdU5TNHhMUzR4SURFdU55MHhOQzQySURRdU9TMHpMakl0TVM0M0lERTBMak1nTVRVdU9DQTJMak10TlM0eUlETXVOQzB4TVM0NUxUUXVOeTB1TkNBekxqWWdOQzR4SURZdU5Ib25MejQ4Y0dGMGFDQmtQU2R0TVRFeU5pNHlJRFV6T1M0MVl6RXVOeUF5TGpjZ05DNDFJRE11TkNBM0xqTWdNUzQyY3pNdU1pMDBMallnTVM0MUxUY3VNMnd0T0M0MUxURXpMakVnTkM0MExUSXVPQ0E0TGpnZ01UTXVOV016SURRdU5pQXhMallnTVRBdE15NDNJREV6TGpRdE5TNDBJRE11TlMweE1DNDRJREl1TlMweE15NDRMVEl1TVd3dE9DNDRMVEV6TGpVZ05DNDBMVEl1T0hvbkx6NDhjR0YwYUNCa1BTZHRNVEUzTWlBMU1qQXVPUzA1TGpFdE1UUXVNUzB1TVM0eElETXVNeUF4Tnk0MkxUSXVPQ0F4TGpndE1UUXVOeTB4TUM0eUxTNHhMakVnT1M0eElERTBMakV0TXk0NUlESXVOUzB4TXk0MkxUSXdMamtnTlM0eExUTXVNeUF4TlM0MklERXhJQzR4TFM0eExUTXVOeTB4T0M0M0lEVXVNUzB6TGpNZ01UTXVOaUF5TUM0NWVpY3ZQanh3WVhSb0lHUTlKMjB4TVRrekxqWWdOVEEyTGprdE1UTXVPU0E1TFRFekxqWXRNakF1T1NBeE15NDVMVGtnTWk0eklETXVOaTA1TGpVZ05pNHhJRE11TXlBMUlEZ3VPUzAxTGpnZ01pNHlJRE11TkMwNExqa2dOUzQ0SURNdU5DQTFMak1nT1M0MUxUWXVNWG9uTHo0OGNHRjBhQ0JrUFNkdE1USXhOUzR6SURRNU1pNDVMVGt1TVMweE5DNHhMUzR4TGpFZ015NHpJREUzTGpZdE1pNDRJREV1T0MweE5DNDNMVEV3TGpJdExqRXVNU0E1TGpFZ01UUXVNUzB6TGprZ01pNDFMVEV6TGpZdE1qQXVPU0ExTGpFdE15NHpJREUxTGpZZ01URWdMakV0TGpFdE15NDNMVEU0TGpjZ05TNHhMVE11TXlBeE15NDJJREl3TGpsNkp5OCtQSEJoZEdnZ1pEMG5iVEV5TWpNZ05EZzNMamt0TVRNdU5pMHlNQzQ1SURrdU1TMDFMamxqTXk0NUxUSXVOU0EzTGpjdE1pNHhJRGt1T0NBeExqRWdNUzQxSURJdU15QXhJRFV1TXkweElEY3VNV3d1TVM0eFl6TXRNUzQySURZdU5DMHVPU0E0TGpJZ01TNDVJREl1TkNBekxqY2dNUzR4SURjdU9DMHpMalFnTVRBdU4zcHRMVE11TlMweE5TNHhJRE11TVMweVl6SXVNeTB4TGpVZ015MHpMalFnTVM0NUxUVXVNaTB4TGpFdE1TNDNMVEl1T1MweExqa3ROQzQ1TFM0MmJDMHpMallnTWk0emVtMDVMalVnTmk0MVl6SXVOUzB4TGpZZ015NHlMVE11TmlBeExqa3ROUzQyY3kwekxqUXRNaTR4TFRVdU9TMHVOR3d0TXk0M0lESXVOQ0F6TGprZ05pNHhlaWN2UGp4d1lYUm9JR1E5SjIweE1qVTJMaklnTkRZMkxqUXRNVE11T1NBNUxURXpMall0TWpBdU9TQXhNeTQ1TFRrZ01pNHpJRE11TmkwNUxqVWdOaTR4SURNdU15QTFJRGd1T1MwMUxqZ2dNaTR5SURNdU5DMDRMamtnTlM0NElETXVOQ0ExTGpNZ09TNDFMVFl1TVhvbkx6NDhjR0YwYUNCa1BTZHRNVEkyTkNBME5qRXVNeTAwTGpRZ01pNDRMVEV6TGpZdE1qQXVPU0E0TGpZdE5TNDJZelF1T0MwekxqRWdPUzR5TFRJdU5DQXhNUzQ1SURFdU9DQXhMamNnTWk0M0lERXVPQ0ExTGprdU1TQTRMalJzT1M0NElEVXVOUzAxSURNdU1pMDRMamt0TlM0eExUTXVOU0F5TGpONmJTMDNMakV0TVRBdU9DQXpMamd0TWk0MFl6SXVNaTB4TGpRZ01pNDNMVE11TlNBeExqTXROUzQyTFRFdU15MHlMVE11TlMweUxqVXROUzQzTFRGc0xUTXVOeUF5TGpSNkp5OCtQSEJoZEdnZ1pEMG5iVEV5TnpndU1TQTBORE11T0dNeExqTWdNUzQySURNdU9TQXhMallnTmk0eUxqRnpNeTR5TFRNdU55QXlMakl0TlM0eVl5MHVPUzB4TGpRdE1pNDBMVEV1TlMwMUxqTXRMalJzTFRNdU1TQXhMakpqTFRRdU5DQXhMamN0Tnk0Mkxqa3RPUzQxTFRJdU1TMHlMalV0TXk0NUxTNDRMVGd1TmlBekxqa3RNVEV1TmlBMExqa3RNeTR5SURrdU5pMHlMallnTVRJdU1TQXhMakZzTFRRdU1TQXlMamRqTFRFdU15MHhMamN0TXk0MUxURXVOeTAxTGpjdExqTnpMVEl1T1NBekxqUXRNUzQ1SURRdU9XTXVPQ0F4TGpNZ01pNHpJREV1TkNBMUxqRXVOR3d5TGprdE1TNHhZelF1T0MweExqZ2dOeTQ0TFRFdU1pQTVMamdnTWlBeUxqWWdOQzR4SURFZ09DNDRMVFF1TWlBeE1pNHhMVFVnTXk0ekxUa3VPQ0F5TGprdE1USXVOUzB1T1hvbkx6NDhjR0YwYUNCa1BTZHRNVE14TVM0eklEUXpNQzQzTFRVdU55MDRMamd0T1M0MElEWXVNU0ExTGpjZ09DNDRMVFF1TkNBeUxqZ3RNVE11TmkweU1DNDVJRFF1TkMweUxqZ2dOUzQxSURndU5TQTVMalF0Tmk0eExUVXVOUzA0TGpVZ05DNDBMVEl1T0NBeE15NDJJREl3TGpsNkp5OCtQSEJoZEdnZ1pEMG5iVEV6TVRrdU5TQTBNalV1TkMweE15NDJMVEl3TGprZ05DNDBMVEl1T0NBeE15NDJJREl3TGpsNkp5OCtQSEJoZEdnZ1pEMG5iVEV6TWpJdU5pQXpPVE11Tm1NMExqTXRNaTQ0SURrdU1TMHhMamdnTVRFdU9TQXlMalZ6TVM0MklEa3VNUzB5TGpnZ01USnNMVE11T1NBeUxqVWdOQzR6SURZdU5pMDBMalFnTWk0NExURXpMall0TWpBdU9YcHRNeUF4TXk0MklETXRNUzQ1WXpJdU5TMHhMallnTXk0eExUTXVPU0F4TGpZdE5pNHpjeTB6TGprdE1pNDRMVFl1TkMweExqRnNMVE1nTVM0NWVpY3ZQanh3WVhSb0lHUTlKMjB4TURFMExqUWdNVEU1Tnk0eExUTXVPQ0F5TGpVdE1URXVPQzB4T0M0eUlEY3VOUzAwTGpoak5DNHlMVEl1TnlBNExUSXVNU0F4TUM0eklERXVOU0F4TGpVZ01pNHpJREV1TmlBMUxqRWdNQ0EzTGpOc09DNDFJRFF1TnkwMExqTWdNaTQ0TFRjdU55MDBMalV0TXk0eElESjZiUzAyTGpFdE9TNDBJRE11TXkweUxqRmpNUzQ1TFRFdU1pQXlMak10TXk0eElERXVNaTAwTGpsekxUTXVNUzB5TGpFdE5TMHVPV3d0TXk0eUlESXVNWG9uTHo0OGNHRjBhQ0JrUFNkdE1UQXpPQzR4SURFeE5qZ3VPR016TGpnZ05TNDRJREl1TnlBeE1TNDJMVEl1TnlBeE5TNHhjeTB4TVM0eElESXVNUzB4TkM0NUxUTXVOeTB5TGpjdE1URXVOaUF5TGpjdE1UVXVNU0F4TVM0eExUSXVNU0F4TkM0NUlETXVOM3B0TFRFekxqY2dPQzQ1WXpJdU5TQXpMamdnTlM0NUlEVWdPQzQ1SURNdU1YTXpMak10TlM0Mkxqa3RPUzQwWXkweUxqVXRNeTQ0TFRZdE5TMDRMamt0TXk0eExUTWdNUzQ1TFRNdU15QTFMall0TGprZ09TNDBlaWN2UGp4d1lYUm9JR1E5SjIweE1EVTRMalFnTVRFMk9DNDJMVEV4TGpnZ055NDJMVEV4TGpndE1UZ3VNaUF6TGpndE1pNDFJRGt1T0NBeE5TNHhJRGd0TlM0eWVpY3ZQanh3WVhSb0lHUTlKMjB4TURjeUxqVWdNVEUxT1M0MUxURXlMakVnTnk0NExURXhMamd0TVRndU1pQXhNaTR4TFRjdU9DQXlJRE11TVMwNExqTWdOUzR6SURJdU9DQTBMalFnTnk0NExUVWdNUzQ1SURJdU9TMDNMamdnTlNBeklEUXVOaUE0TGpNdE5TNHplaWN2UGp4MFpYaDBJR1p2Ym5RdFptRnRhV3g1UFNkVFJsQnliMFJwYzNCc1lYa3RVbVZuZFd4aGNpY2dabTl1ZEMxemFYcGxQU2N6Tmk0M05qUTNKeUJzWlhSMFpYSXRjM0JoWTJsdVp6MG5NU2NnZEhKaGJuTm1iM0p0UFNkdFlYUnlhWGdvTGpnek9UTWdMUzQxTkRNMklDNDFORE0ySUM0NE16a3pJREV3TXpRdU5qRTVPU0F4TWpNNUxqZzJOVElwSno1UWIyUmpZWE4wSUVKbFoybHVibVZ5UEM5MFpYaDBQanh3WVhSb0lHUTlKMjA0TlRRdU5pQTVOVEl1TW1NeExqSWdNUzQwSURNdU5DQXhMalFnTlM0MExqRnpNaTQ0TFRNdU1pQXhMamt0TkM0MVl5MHVPQzB4TGpJdE1pNHhMVEV1TXkwMExqWXRMak5zTFRJdU55QXhZeTB6TGpnZ01TNDFMVFl1Tmk0NExUZ3VNeTB4TGpndE1pNHlMVE11TkMwdU55MDNMalVnTXk0MExURXdMakVnTkM0eUxUSXVOeUE0TGpNdE1pNHpJREV3TGpVZ01Xd3RNeTQySURJdU0yTXRNUzR4TFRFdU5TMHpMakV0TVM0MUxUVXRMak56TFRJdU5pQXlMamt0TVM0M0lEUXVNMk11TnlBeExqRWdNaUF4TGpJZ05DNDBMak5zTWk0MUxURmpOQzR5TFRFdU5pQTJMamd0TVNBNExqWWdNUzQzSURJdU15QXpMalV1T1NBM0xqWXRNeTQySURFd0xqVXROQzQwSURJdU9DMDRMalVnTWk0MkxURXdMamd0TGpoNkp5OCtQSEJoZEdnZ1pEMG5iVGc0TXk0MUlEazBNQzQ0TFRVdE55NDNMVGd1TWlBMUxqTWdOU0EzTGpjdE15NDRJREl1TlMweE1TNDRMVEU0TGpJZ015NDRMVEl1TlNBMExqZ2dOeTQwSURndU1pMDFMak10TkM0NExUY3VOQ0F6TGpndE1pNDFJREV4TGpnZ01UZ3VNbm9uTHo0OGNHRjBhQ0JrUFNkdE9UQXhMallnT1RFMkxqSmpNeTQ0SURVdU9DQXlMamNnTVRFdU5pMHlMamNnTVRVdU1YTXRNVEV1TVNBeUxqRXRNVFF1T1MwekxqY3RNaTQzTFRFeExqWWdNaTQzTFRFMUxqRmpOUzQwTFRNdU5pQXhNUzR5TFRJdU1pQXhOQzQ1SURNdU4zcHRMVEV6TGpjZ09DNDRZekl1TlNBekxqZ2dOUzQ1SURVZ09DNDVJRE11TVhNekxqTXROUzQyTGprdE9TNDBZeTB5TGpVdE15NDRMVFl0TlMwNExqa3RNeTR4TFRNZ01TNDVMVE11TXlBMUxqWXRMamtnT1M0MGVpY3ZQanh3WVhSb0lHUTlKMjA1TVRZdU9DQTVNVGt1TWkwekxqWWdNaTR6TFRFMkxqWXRNVFV1TVNBekxqa3RNaTQySURFeExqUWdNVEV1TXk0eExTNHhMVFV1TVMweE5TNDBJRE11TWkweUxqRWdNVEV1T1NBeE1TQXVNUzB1TVMwMUxqY3RNVFV1TVNBekxqa3RNaTQySURjZ01qRXVNeTB6TGpZZ01pNHpMVEV4TGpVdE1UQXVNaTB1TVM0eGVpY3ZQanh3WVhSb0lHUTlKMjA1TXprdU5TQTVNRFF1TmkweE1TNDRMVEU0TGpJZ015NHlMVEl1TVNBeE5pNHlJRFl1TkM0eExTNHhMVGN1TnkweE1TNDVJRE11TmkweUxqUWdNVEV1T0NBeE9DNHlMVE11TWlBeUxURTJMakl0Tmk0MExTNHhMakVnTnk0M0lERXhMamw2Snk4K1BIQmhkR2dnWkQwbmJUazJOUzQySURnNE1TNDBMVFl1TkNBMExqRWdNUzQxSURVdU15MHpMamtnTWk0MUxUVXVOUzB5TWk0eklEUXVOUzB5TGprZ01UZ3VNU0F4TkM0eExUUXVNU0F5TGpaNmJTMDNMalF1TnlBMExqZ3RNeTR4TFRjdU1pMDFMamt0TGpFdU1Yb25MejQ4Y0dGMGFDQmtQU2R0T1RreExqY2dPRGN3TGpjdE9DMHhNaTR6TFM0eExqRWdNaTQ1SURFMUxqTXRNaTQwSURFdU5pMHhNaTQ0TFRndU9TMHVNUzR4SURnZ01USXVNeTB6TGpRZ01pNHlMVEV4TGpndE1UZ3VNeUEwTGpRdE1pNDVJREV6TGpVZ09TNDJMakV0TGpFdE15NHlMVEUyTGpNZ05DNDBMVEl1T1NBeE1TNDRJREU0TGpKNkp5OCtQSEJoZEdnZ1pEMG5iVEV3TVRBdU5TQTROVGd1TmkweE1pNHhJRGN1T0MweE1TNDRMVEU0TGpJZ01USXVNUzAzTGpnZ01pQXpMakV0T0M0eklEVXVNeUF5TGpnZ05DNDBJRGN1T0MwMUlERXVPU0F5TGprdE55NDRJRFVnTXlBMExqWWdPQzR6TFRVdU0zb25MejQ4ZEdWNGRDQm1iMjUwTFdaaGJXbHNlVDBuVTBaUWNtOUVhWE53YkdGNUxWSmxaM1ZzWVhJbklHWnZiblF0YzJsNlpUMG5Nell1TnpZME55Y2diR1YwZEdWeUxYTndZV05wYm1jOUp6RW5JSFJ5WVc1elptOXliVDBuYldGMGNtbDRLQzQ0TXpreklDMHVOVFF6TmlBdU5UUXpOaUF1T0RNNU15QTROemd1T1RnNE15QTVPVGt1TlRjeU1Ta25Qa3BQU1NCSlZFOGdVRTlFUTBGVFZEd3ZkR1Y0ZEQ0OGNHRjBhQ0JrUFNkdE1USXdNeTR4SURjd09HTXpMamN0TWk0MElEY3VPUzB4TGpZZ01UQXVNeUF5TGpGek1TNDBJRGd0TWk0MUlERXdMalJzTFRNdU5DQXlMaklnTXk0M0lEVXVOeTB6TGpnZ01pNDFMVEV4TGpndE1UZ3VNbnB0TWk0MUlERXhMamdnTWk0MkxURXVOMk15TGpJdE1TNDBJREl1TnkwekxqUWdNUzQwTFRVdU5YTXRNeTQwTFRJdU5DMDFMalV0TVd3dE1pNDJJREV1TjNvbkx6NDhjR0YwYUNCa1BTZHRNVEl6TkM0MklEY3dNQzQxWXpNdU9DQTFMamdnTWk0M0lERXhMall0TWk0M0lERTFMakZ6TFRFeExqRWdNaTR4TFRFMExqa3RNeTQzTFRJdU55MHhNUzQySURJdU55MHhOUzR4SURFeExqRXRNaTR4SURFMExqa2dNeTQzZW0wdE1UTXVOeUE0TGpsak1pNDFJRE11T0NBMUxqa2dOU0E0TGprZ015QXpMVEV1T1NBekxqTXROUzQyTGprdE9TNDBMVEl1TlMwekxqZ3ROUzQ1TFRVdE9DNDVMVE11TVMweklESXRNeTQwSURVdU55MHVPU0E1TGpWNkp5OCtQSEJoZEdnZ1pEMG5iVEV5TkRNdU1TQTNNRGN1T1MweE1TNDRMVEU0TGpJZ015NDRMVEl1TlNBeE1TNDRJREU0TGpKNkp5OCtQSEJoZEdnZ1pEMG5iVEV5TlRBdU1pQTNNRE11TXkweE1TNDRMVEU0TGpJZ015NHlMVEl1TVNBeE5pNHlJRFl1TkM0eExTNHhMVGN1TnkweE1TNDVJRE11TmkweUxqUWdNVEV1T0NBeE9DNHlMVE11TWlBeUxURTJMakl0Tmk0MExTNHhMakVnTnk0M0lERXhMamw2Snk4K1BIQmhkR2dnWkQwbmJURXlOekl1T1NBMk9EZ3VOaTA1TGpndE1UVXVNUzAxTGpNZ015NDBMVEl0TXk0eElERTBMak10T1M0eklESWdNeTR4TFRVdU15QXpMalFnT1M0NElERTFMakY2Snk4K1BIUmxlSFFnWm05dWRDMW1ZVzFwYkhrOUoxTkdVSEp2UkdsemNHeGhlUzFTWldkMWJHRnlKeUJtYjI1MExYTnBlbVU5SnpNMkxqYzJORGNuSUd4bGRIUmxjaTF6Y0dGamFXNW5QU2N4SnlCMGNtRnVjMlp2Y20wOUoyMWhkSEpwZUNndU9ETTVNeUF0TGpVME16WWdMalUwTXpZZ0xqZ3pPVE1nTVRJek1TNDBPRGc0SURjM01TNHlOakl5S1NjK0pERXdTR1Z1YTJGcmRUd3ZkR1Y0ZEQ0OGNHRjBhQ0JrUFNkdE9USTRMalFnTVRBMU5pNDBZek11TnkweUxqUWdOeTQ1TFRFdU5pQXhNQzR6SURJdU1YTXhMalFnT0MweUxqVWdNVEF1Tkd3dE15NDBJREl1TWlBekxqY2dOUzQzTFRNdU9DQXlMalV0TVRFdU9DMHhPQzR5ZW0weUxqVWdNVEV1T1NBeUxqWXRNUzQzWXpJdU1pMHhMalFnTWk0M0xUTXVOQ0F4TGpRdE5TNDFjeTB6TGpRdE1pNDBMVFV1TlMweGJDMHlMallnTVM0M2VpY3ZQanh3WVhSb0lHUTlKMjA1TlRJdU5DQXhNRFkyTGpndE15NDRJREl1TlMweE1TNDRMVEU0TGpJZ055NDFMVFF1T0dNMExqSXRNaTQzSURndE1pNHhJREV3TGpNZ01TNDFJREV1TlNBeUxqTWdNUzQySURVdU1TQXdJRGN1TTJ3NExqVWdOQzQzTFRRdU15QXlMamd0Tnk0M0xUUXVOUzB6TGpFZ01ucHRMVFl1TVMwNUxqVWdNeTR6TFRJdU1XTXhMamt0TVM0eUlESXVNeTB6TGpFZ01TNHlMVFF1T1hNdE15NHhMVEl1TVMwMUxTNDViQzB6TGpJZ01pNHhlaWN2UGp4d1lYUm9JR1E5SjIwNU56WXVNU0F4TURNNExqVmpNeTQ0SURVdU9DQXlMamNnTVRFdU5pMHlMamNnTVRVdU1YTXRNVEV1TVNBeUxqRXRNVFF1T1MwekxqY3RNaTQzTFRFeExqWWdNaTQzTFRFMUxqRmpOUzQwTFRNdU5pQXhNUzR4TFRJdU1TQXhOQzQ1SURNdU4zcHRMVEV6TGpjZ09DNDRZekl1TlNBekxqZ2dOUzQ1SURVZ09DNDVJRE11TVhNekxqTXROUzQyTGprdE9TNDBZeTB5TGpVdE15NDRMVFl0TlMwNExqa3RNeTR4Y3kwekxqUWdOUzQyTFM0NUlEa3VOSG9uTHo0OGNHRjBhQ0JrUFNkdE9UZzRMallnTVRBME15NHpMVEUzTGprdE1UUXVNaUEwTGpNdE1pNDRJREV6TGpRZ01URXVOaTR4TFM0eExUVXVNUzB4TnlBMExqSXRNaTQzSURVdU55QXlNaTR5ZWljdlBqeHdZWFJvSUdROUoyMHhNREV6TGpJZ01UQXlOeTR6TFRFeUxqRWdOeTQ0TFRFeExqZ3RNVGd1TWlBeE1pNHhMVGN1T0NBeUlETXVNUzA0TGpNZ05TNHpJREl1T0NBMExqUWdOeTQ0TFRVZ01TNDVJREl1T1MwM0xqZ2dOU0F6SURRdU5pQTRMak10TlM0emVpY3ZQanh3WVhSb0lHUTlKMjB4TURFeExqUWdNVEF3TWk0M1l6VXVOUzB6TGpVZ01UQXVPQzB5TGpRZ01UUXVOU0F6TGpSek1pNDNJREV4TGpNdE1pNDNJREUwTGpoc0xUY2dOQzQxTFRFeExqZ3RNVGd1TW5wdE5pNDJJREUzTGpFZ01pNDNMVEV1TjJNekxqUXRNaTR5SURNdU9TMDFMalVnTVM0MExUa3VOQzB5TGpVdE15NDRMVFV1TnkwMExqY3RPUzR4TFRJdU5Xd3RNaTQzSURFdU4zb25MejQ4Y0dGMGFDQmtQU2R0TVRBek9TNDBJREV3TVRBdU15MHhNUzQ0TFRFNExqSWdOeTQ1TFRVdU1XTXpMalF0TWk0eUlEWXVOeTB4TGprZ09DNDFMamtnTVM0eklESWdMamtnTkM0MkxTNDVJRFl1TW13dU1TNHhZekl1TmkweExqUWdOUzQyTFM0NElEY3VNU0F4TGpZZ01pNHhJRE11TWlBeElEWXVPQzB5TGprZ09TNHplbTB0TXkweE15NHhJREl1TnkweExqaGpNaTB4TGpNZ01pNDJMVE1nTVM0MkxUUXVOUzB1T1MweExqVXRNaTQxTFRFdU5pMDBMak10TGpWc0xUTXVNU0F5ZW0wNExqTWdOUzQzWXpJdU1pMHhMalFnTWk0NExUTXVNU0F4TGpZdE5DNDVMVEV1TVMweExqY3RNaTQ1TFRFdU9DMDFMakl0TGpSc0xUTXVNaUF5TGpFZ015NDBJRFV1TTNvbkx6NDhjR0YwYUNCa1BTZHRNVEEyTUM0MElEazVOaTQ0TFRRdU5DMDJMamd0TVRNdU55MDNMak1nTkM0eUxUSXVOeUE1TGpFZ05TNHhMakV0TGpFdExqa3RNVEF1TkNBMExqRXRNaTQySURFZ01UVXVOU0EwTGpRZ05pNDRlaWN2UGp4MFpYaDBJR1p2Ym5RdFptRnRhV3g1UFNkVFJsQnliMFJwYzNCc1lYa3RVbVZuZFd4aGNpY2dabTl1ZEMxemFYcGxQU2N6Tmk0M05qUTNKeUJzWlhSMFpYSXRjM0JoWTJsdVp6MG5NU2NnZEhKaGJuTm1iM0p0UFNkdFlYUnlhWGdvTGpnek9UTWdMUzQxTkRNMklDNDFORE0ySUM0NE16a3pJRGsxTmk0NE1EVXlJREV4TVRrdU56RTROU2tuUGtobGJtdGhhM1VnUTI5dGJYVnVhWFI1UEM5MFpYaDBQanh3WVhSb0lHUTlKMjB4TWpnekxqSWdPREkyTGpaak5TNDFMVE11TlNBeE1DNDRMVEl1TkNBeE5DNDFJRE11TkhNeUxqY2dNVEV1TXkweUxqY2dNVFF1T0d3dE55QTBMalV0TVRFdU9DMHhPQzR5ZW0wMkxqWWdNVGN1TVNBeUxqY3RNUzQzWXpNdU5DMHlMaklnTXk0NUxUVXVOU0F4TGpRdE9TNDBMVEl1TlMwekxqZ3ROUzQzTFRRdU55MDVMakV0TWk0MWJDMHlMamNnTVM0M2VpY3ZQanh3WVhSb0lHUTlKMjB4TXpFeklEZ3lOaTQ1TFRZdU5DQTBMakVnTVM0MUlEVXVNeTB6TGprZ01pNDFMVFV1TlMweU1pNHpJRFF1TlMweUxqa2dNVGd1TVNBeE5DNHhMVFF1TVNBeUxqWjZiUzAzTGpRdU9DQTBMamd0TXk0eExUY3VNaTAxTGprdExqRXVNWG9uTHo0OGNHRjBhQ0JrUFNkdE1UTXlOUzQwSURneU5TNHhMVGt1T0MweE5TNHhMVFV1TXlBekxqUXRNaTB6TGpFZ01UUXVNeTA1TGpNZ01pQXpMakV0TlM0eklETXVOQ0E1TGpnZ01UVXVNWG9uTHo0OGNHRjBhQ0JrUFNkdE1UTTBPQzQySURneE1DNHhMVEV5TGpFZ055NDRMVEV4TGpndE1UZ3VNaUF4TWk0eExUY3VPQ0F5SURNdU1TMDRMak1nTlM0eklESXVPQ0EwTGpRZ055NDRMVFVnTVM0NUlESXVPUzAzTGpnZ05TQXpJRFF1TmlBNExqTXROUzR6ZWljdlBqeDBaWGgwSUdadmJuUXRabUZ0YVd4NVBTZFRSbEJ5YjBScGMzQnNZWGt0VW1WbmRXeGhjaWNnWm05dWRDMXphWHBsUFNjek5pNDNOalEzSnlCc1pYUjBaWEl0YzNCaFkybHVaejBuTVNjZ2RISmhibk5tYjNKdFBTZHRZWFJ5YVhnb0xqZ3pPVE1nTFM0MU5ETTJJQzQxTkRNMklDNDRNemt6SURFek1USXVNRGs0TkNBNE9Ea3VOakF5TWlrblBqSXdNakl1TURJdU1UVThMM1JsZUhRK1BIUmxlSFFnWm05dWRDMW1ZVzFwYkhrOUowaGxiSFpsZEdsallTMUNiMnhrSnlCbWIyNTBMWE5wZW1VOUp6VTVMamc0T1RJbklIUnlZVzV6Wm05eWJUMG5iV0YwY21sNEtDNDRNemt6SUMwdU5UUXpOaUF1TlRRek5pQXVPRE01TXlBM01Ea3VPVFEyTWlBeE16azVMalEyTnpjcEp6NXBjMkowZEhrOEwzUmxlSFErUEM5blBqeG5JR05zYVhBdGNHRjBhRDBuZFhKc0tDTmhLU2MrUEdsdFlXZGxJR2hsYVdkb2REMG5OREkySnlCdmRtVnlabXh2ZHowbmRtbHphV0pzWlNjZ2RISmhibk5tYjNKdFBTZHRZWFJ5YVhnb0xqVXdNemNnTFM0ek1qWXlJQzR6TWpZeUlDNDFNRE0zSURVeE5TNDNOemN6SURFeE5EUXVOamN5TnlrbklIZHBaSFJvUFNjME1qWW5JR2h5WldZOUoyaDBkSEJ6T2k4dmJHZ3pMbWR2YjJkc1pYVnpaWEpqYjI1MFpXNTBMbU52YlM5UldWSjBVM0p4TmsxMlJIZHlNRmw1TFZBdE1YQjZhbE0wVnpWNFpXWm9Uall4VlZsVWRHaG1abFJTTmxoSWRGSkdhVzloVkdOMmRYcDRiRFJRYmpoZlYzcHNMVmhaTmxscFdsb3ROVmRDVkVvd2FYb3dVMjFCWTJscVRYUTJTVEJJTWpaWk4ybHpQWGMyTURBbkx6NDhMMmMrUEdjZ1ptbHNiRDBuSTJabVppYytQSFJsZUhRZ1ptOXVkQzFtWVcxcGJIazlKMU5HVUhKdlJHbHpjR3hoZVMxU1pXZDFiR0Z5SnlCbWIyNTBMWE5wZW1VOUp6TXpMakU0T0RjbklHeGxkSFJsY2kxemNHRmphVzVuUFNjeEp5QjBjbUZ1YzJadmNtMDlKMjFoZEhKcGVDZ3VPRE01TXlBdExqVTBNellnTGpVME16WWdMamd6T1RNZ056azFMakE0TlNBeE5ETTJMalEyTmprcEp6NHdlR016TGk0dVEwWmxQQzkwWlhoMFBqeHdZWFJvSUdROUoyMDNOekV1TXlBeE5ESTNMalF0TVM0M0lERXVNUzB4TGpndE1pNDNZeTB4TGpjdE1pNDJMVFV1TVMwekxqTXROeTQzTFRFdU4yd3RNVGd1TnlBeE1pNHhZeTB5TGpZZ01TNDNMVE11TXlBMUxqRXRNUzQzSURjdU4yd3hNaTR4SURFNExqZGpNUzQzSURJdU5pQTFMakVnTXk0eklEY3VOeUF4TGpkc01UZ3VOeTB4TWk0eFl6SXVOaTB4TGpjZ015NHpMVFV1TVNBeExqY3ROeTQzYkMweExqZ3RNaTQzSURFdU55MHhMakZqTVM0M0xURXVNU0F5TGpJdE15NDFJREV1TVMwMUxqSnNMVFF1TlMwMkxqbGpMVEV0TVM0NUxUTXVNeTB5TGpRdE5TNHhMVEV1TW5wdE5TNHlJREU1TGpOakxqUXVOeTR5SURFdU5pMHVOQ0F5YkMweE9DNDNJREV5TGpGakxTNDNMalF0TVM0MkxqSXRNaTB1Tkd3dE1USXVNUzB4T0M0M1l5MHVOQzB1TnkwdU1pMHhMall1TkMweWJERTRMamN0TVRJdU1XTXVOeTB1TkNBeExqWXRMaklnTWlBdU5Hd3hMamdnTWk0M0xUUXVNeUF5TGpoakxURXVOeUF4TGpFdE1pNHlJRE11TlMweExqRWdOUzR5YkRRdU5TQTJMamxqTVM0eElERXVOeUF6TGpVZ01pNHlJRFV1TWlBeExqRnNOQzR6TFRJdU9IcHRMamd0T1M0MExURXVOQzQ1TFRNdU5TQXlMak10TkNBeUxqWXROQzR4TFRZdU15QTBMVEl1TmlBekxqVXRNaTR6SURFdU5DMHVPWG9uTHo0OGNHRjBhQ0JrUFNkdE56WTRMaklnTVRRek5pNDBZeTB1T1M0MkxURXVNaUF4TGpndExqWWdNaTQ0TGpZdU9TQXhMamdnTVM0eUlESXVPQzQyY3pFdU1pMHhMamd1TmkweUxqaGpMUzQyTFM0NUxURXVPUzB4TGpJdE1pNDRMUzQyZWljdlBqd3ZaejQ4TDJjK1BDOXpkMmwwWTJnK1BDOXpkbWMrIn0=
```
<img width="696" alt="スクリーンショット 2022-02-19 23 25 26" src="https://user-images.githubusercontent.com/701242/154804921-04f77d9c-d70b-47bd-8550-bc8103dce658.png">

In the image section, data:image/svg+xml;base64 should be added.

Reply:
seems like ok, so maybe problem lays on svg itself. 
can you try another simple svg image ? we it work then we need to fix svg

Reply:
Yes! The problem is with the external image reference in svg. For example, the Henkaku logo below can be used by itself.

```
<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 510.4 512">
  <path d="M421.62,426.57h17.67v-5.24h19.87v5.24h19v-5.24H498v5.24h17.27v12.92H498v13.73H477.84v3.63h31.48v13.68a15.81,15.81,0,0,1-15.8,15.8H477.84V490h37.44v12.92H477.84v12.51h-18V502.88H422.42V490h37.44v-3.63H428V456.85h31.87v-3.63H439.29V439.49H421.62Zm26.18,47.24h12.06v-4.44H447.8Zm11.36-31.49h19v-2.83h-19Zm18.68,31.49h11.6v-4.44h-11.6Zm-31.36-136-12.11-6.86q-3.24,8.47-14.54,14.53l9.29,11.71Q440,351.12,446.48,337.8Zm48.45,40.37c4-3.23,5.78-4.69,7.39-6.57V362H465.86l3.23-3.63h9.69a7.68,7.68,0,0,0,6.46-3.23,10.2,10.2,0,0,0,2.42-6.86v-19H513.5V316.4H474.74v-6.86H459.4v6.86H421.85v12.92H449.3q.4,8.08-1.61,13.33a26.68,26.68,0,0,1-8.88,10.9l8.88,7.67a39.55,39.55,0,0,1-9.29,6.46,76,76,0,0,1-16.48,6.42v12.77A56.2,56.2,0,0,0,434.48,384a84.17,84.17,0,0,0,9.17-4.22,51.27,51.27,0,0,0,8.48,6q-4,2-13.73,3.64a122.71,122.71,0,0,1-16.48,1.33v13.73a147.91,147.91,0,0,0,27.79-3.35,85.17,85.17,0,0,0,19.38-6.87c12.2,6.71,28.25,9.19,44.4,9.46V389.07c-13.19,0-22.06-1.34-28.25-4A61.74,61.74,0,0,0,494.93,378.17Zm-30.69-48.85h9.29v11.31s.18,3.63-3.23,3.63h-6.46l3.23,12.92L459,355.57Q463,349.92,464.24,329.32Zm10.5,47.24c-2.69,1.88-4.71,2.83-6.06,2.83a24.29,24.29,0,0,1-10.49-6.46h21.39A36.78,36.78,0,0,1,474.74,376.56ZM501,331.34l-11.71,8.08q3.24,2,8.08,8.47a35.62,35.62,0,0,1,6,9.29l11.71-9.29a59.5,59.5,0,0,0-6.46-8.88A50.23,50.23,0,0,0,501,331.34ZM172.45,419.78a42.73,42.73,0,0,0-30,11.55C134.16,439,130,448.47,130,459.41v54.81h23.71V493.33h37.54v20.89h23.71V459.41c0-10.94-4.19-20.39-12.46-28.08A42.73,42.73,0,0,0,172.45,419.78Zm18.77,51.27H153.68V459.41a16,16,0,0,1,5.4-12.28,20.19,20.19,0,0,1,26.74,0,16,16,0,0,1,5.4,12.28Zm122-49.34-33.77-.17-27.05,36.62V421.68H228.71v92.54h23.71V474.08l31.12,40.28,33.9-.16-40.56-48.14Zm72.48,0V476.5a16,16,0,0,1-5.4,12.27,20.18,20.18,0,0,1-26.75,0,16,16,0,0,1-5.39-12.27V421.68H324.47V476.5c0,10.93,4.19,20.37,12.46,28.07a44.78,44.78,0,0,0,60,0c8.27-7.7,12.46-17.14,12.46-28.07V421.68ZM347.4,403.76v-51l42.78,51,18.46,0V316.59L4.89,4.12V513.77l30.54-40.4v40.85H59.14V474.08l31.12,40.28,33.9-.16L83.6,466.06,120,421.71l-33.77-.17-27,36.62V442l71.47-94.53v56.29H153.9V368.63h38v35.13h23.28v-87.7l22.71.06v87.64H302V381.9H261.14V368.63H302V346.78H261.14V333.51H302V316.27l22.14.06v87.43Zm38-41.09-38.84-46.29,38.84.1Zm-193.5-15.89h-38V316.66l.56-.75,37.4.09Z" transform="translate(-4.89 -4.12)"/>
</svg>
```

The two images below that are problematic are the two that are difficult to vectorize. Also, if you embed them, you will run into the problem of image size.

![unnamed](https://user-images.githubusercontent.com/701242/154832719-3f2fa6d1-65a2-441d-8bf4-348b8459a975.png)
![QmUETsUz1cR6pi9LPyEFnGrcwQYoGtvZmDdzZ2GdL4kfwv](https://user-images.githubusercontent.com/701242/154832722-5d384313-c3e3-4e66-af50-6ec04f9451f9.jpg)

Reply:
i see so why not modify structure of svg ?
maybe just try to use image tag instead of foreign object
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image

Reply:
If I understand the instructions correctly, the following is what we have implemented Unfortunately, this one did not show up either.

https://github.com/henkaku-center/podcast-nft/pull/6/commits/42abe3814dbb784b7c3d71f2718dd78b6d6f690d